doc/source/configuration.rst

```diff
@@ -55,9 +55,7 @@ The analysis configuration file is composed of a first section defining the spec
 
 The [SPECIES] section includes:
 
-.. TODO: are spaces tolerated in species name?
-
-* **species**: name of the focal species. A *K*:sub:`S` paralog distribution is generated for this species and its *K*:sub:`S`-scale is used as the rate-adjustment reference. It is advised to use a short name (for example, the genus or family name) or common abbreviation here. For example, "elaeis" or "eguineensis" instead of "Elaeis guineensis".
+* **species**: name of the focal species. A *K*:sub:`S` paralog distribution is generated for this species and its *K*:sub:`S`-scale is used as the rate-adjustment reference. The name must be composed of a single word since spaces are not tolerated; it is advised to use a short name (for example, the genus or family name) or common abbreviation here. For example, "elaeis" or "eguineensis" instead of "Elaeis guineensis".
```

Comment:
```suggestion
* **species**: name of the focal species. A *K*:sub:`S` paralog distribution is generated for this species and its *K*:sub:`S`-scale is used as the rate-adjustment reference. It is advised to use a short name (for example, the genus or family name) or common abbreviation here, spaces are not tolerated. For example, "elaeis" or "eguineensis" instead of "Elaeis guineensis".
```